src/Orchard.Web/Modules/Orchard.Setup/Services/SetupService.cs

```diff
@@ -154,6 +155,10 @@ public class SetupService : ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
 
+                    // Create the distributed lock record schema.
+                    var distributedLockSchemaBuilder = new DistributedLockSchemaBuilder(_shellSettings, schemaBuilder);

```

Comment:
True, but I removed that migration so we don't have two places to maintain the schema. Do you think it's better to have both a migration _and_ another class that creates the same table?